.travis.yml

```diff
@@ -37,16 +37,17 @@ install:
   - mkdir --parents ~/.config/mapswipe_workers
   - mkdir --parents ~/.local/share/mapswipe_workers
   - echo "$SERVICE_ACCOUNT_KEY" > ~/.config/mapswipe_workers/serviceAccountKey.json
-  - docker build --tag mapswipe_postgres --file postgres/Dockerfile-dev postgres/
+  - docker build --tag mapswipe_postgres --file postgres/Dockerfile-dev postgres/  # Dockerfile-dev is used because backup routine is not needed for travis builds
+  - docker-compose build mapswipe_workers
 
 before_script:
   - docker run --detach --publish 5432:5432 --name mapswipe_postgres -e POSTGRES_DB="$POSTGRES_DB" -e POSTGRES_USER="$POSTGRES_USER" -e POSTGRES_PASSWORD="$POSTGRES_PASSWORD" mapswipe_postgres
   - docker-compose up --build --detach firebase_deploy
```

Comment:
which firebase instance is used here again? `dev` or `test`. and do I still need to setup the `test` firebase correctly?

Reply:
At the moment the test or cli instance of firebase is used.